lib/index.js

```diff
@@ -442,6 +454,10 @@ Builder.prototype.build = function(options) {
 
 			// Compile theme if not cached or RTL is requested and missing in cache
 			if (!themeCache || (options.rtl && !themeCache.result.cssRtl)) {
+				var scopeOptions = options.scope;
+				if (scopeOptions.selector && scopeOptions.embeddedFilePath && scopeOptions.embeddedCompareFilePath && scopeOptions.baseFile) {
+					return compileWithScope(scopeOptions);
```

Comment:
What about `addInlineParameters` and `cacheTheme`? Those won't be called in this case, right?

Reply:
They should probably be added too, you're right.